scripts/release-helm.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
@josvazg not necessarily related to this PR: please add a shell script linter as well in the build process.

Reply:
We have one at the CI level, I would like to move it to be available from the Makefile as well. And have the CI use the same linting code path as the makefile.